sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -40,23 +35,16 @@
                 " /></v-col
```

Comment:
I see that the restriction note is always empty in screenshot. why so ?

![image](https://user-images.githubusercontent.com/54600590/206249145-dcda81d1-4ba6-4b5d-bcb8-3c34019bf93c.png)